src/Goteo/Model/Invest.php

```diff
@@ -492,6 +493,13 @@ public static function getSQLFilter($filters = []) {
                     $sqlFilter[] = "invest.resign = 0";
                     $sqlFilter[] = "invest.campaign = 0";
                     break;
+                case 'from_subscription':
+                    $values[':subscription_methods'] = implode(',', [
+                        StripeSubscriptionPaymentMethod::PAYMENT_METHOD_ID
+                    ]);
+
+                    $sqlFilter[] = "invest.method IN (:subscription_methods)";
```

Comment:
Are we expecting to have more Subscription methods? Since we currently only have one, and the filter is in singular, i would prefer that we check that the value is the filtered value. Afterwards we can add a new filter to handle multiple subsriptions methods.

Also why can't we use the already created 'method' filter? It uses an array as input, so it also works like the one you coded.

We can still approve this PR since it's working correctly.